GUI/sanity-test-halo/jest-puppeteer/jest-puppeteer.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    launch: {
+        headless: true
+    },
+    browserContext: 'default',
```

Comment:
Codacy has a fix for the issue: [Replace `··browserContext:·'default'` with `browserContext:·"default"`](https://app.codacy.com/gh/Seagate/cortx-experiments/pullRequest?prid=9333958)
```suggestion
  browserContext: "default",
```